modules/steamdeck/workarounds.nix

```diff
@@ -1,8 +1,11 @@
 { config, lib, pkgs, ... }:
-
+let
+  superCfg = config.jovian.devices.steamdeck;
+  cfg = config.jovian.devices.steamdeck.workarounds;
+in
 {
   options = {
-    jovian = {
+    jovian.devices.steamdeck = {
```

Comment:
This is not steam deck specific.

Assuming we grow support for other "fun" targets with "optimized" kernel builds, the workaround would possibly be needed too.